hack/ginkgo-e2e.sh

```diff
@@ -99,6 +99,11 @@ elif [[ "${KUBERNETES_PROVIDER}" == "aws" ]]; then
   auth_config=(
     "--auth_config=${HOME}/.kubernetes_auth"
   )
+elif [[ "${KUBERNETES_PROVIDER}" == "conformance_test" ]]; then
```

Comment:
Right. Should skip that and also check for AUTH_CONFIG before this long chain of if/elif on the provider.